src/pages/Dashboard.tsx

```diff
@@ -122,6 +172,38 @@ const Dashboard = () => {
             <Button onClick={createNewInsight}>
                 New Insight
             </Button>
+            {/**TODO: Popup form for new group**/}
+            <div>
+                <form onSubmit={() => createNewGroup(newGroupNameInput)}>
```

Comment:
Forms require some sort of validation, remind me to add either Formik or React Hook Forms